src/schoology-extractor/schoology_extractor/api/request_client.py

```diff
@@ -401,6 +394,39 @@ def get_roles(self, page_size: int = DEFAULT_PAGE_SIZE) -> PaginatedResult:
             RESOURCE_NAMES.ROLE,
             self.base_url + url)
 
-    def _build_query_params_for_first_page(self, page_size: int):
-        assert isinstance(page_size, int), "Argument `page_size` should be of type `int`."
-        return f"start=0&limit={page_size}"
+    def get_enrollments(
+        self, section_id: int, page_size: int = DEFAULT_PAGE_SIZE
+    ) -> list:
+        """
+        Retrieves enrollment data for a section, with support for paging.
+
+        Parameters
+        ----------
+        section_id : int
+            A Section Id
+
```

Comment:
```suggestion
        page_size : int
            Number of items per page
```

Reply:
You merged before I could commit 😆 . I'll update with the next PR.